### ISSUE TITLE
New Term - substrate

### ISSUE THREAD

**Issue by crayfish66** [tdwg/dwc#504]
## New term

* Submitter: Wojciech Paul (w.paul@botany.pl)
* Efficacy Justification (why is this term necessary?): for many organisms (esp. from Fungi s.l. group) substrate is one of basic parameters describing direct environmental conditions; not to be confused with "habitat", that describes 'wider' environment (as forest, meadow, plant community etc.). Implementing the new term would allow to more efficient filtering data on occurrence conditions, biological demands of taxa, preparing lists of taxa preferring given type of substrate etc. With present practice of placing this data into varied fields (see "Stability justification" below) and mixing it with other types of data within them, makes such actions complicated or even impractical.
* Demand Justification (name at least two organizations that independently need this term): RCIN [Digital Repository of Scientific Institutions, Poland] https://rcin.org.pl/ , (this term has been already introduced there due to its lack in DwC); W. Szafer Institute of Botany, Polish Academy of Sciences https://www.botany.pl (to allow compatibility of the internal collections' databases with external repositories using DwC, e.g. GBIF); probably also other institutions, journals etc. dealing with occurrence data of Fungi (incl. lichens, myxomycetes etc.) wanting to implement open data repositories
* Stability Justification (what concerns are there that this might affect existing implementations?): as the observed usage suggests, these data have been input so far inconsistently into varied fields of '...Remarks' term type, or (in case of parasitism) the substrate taxon have been put into associatedTaxa field; efficient usage of the new term, although not obligatory, would demand transferring this piece of data to the newly created field.
* Implications for dwciri: namespace (does this change affect a dwciri term version)?: No dwciri: term would be needed for this term.

Proposed attributes of the new term:

* Term name (in lowerCamelCase for properties, UpperCamelCase for classes): substrate
* Term label (English, not normative): Substrate
* Organized in Class (e.g., Occurrence, Event, Location, Taxon): Occurrence
* Definition of the term (normative): A type of biotic or abiotic material to which a dwc:Organism was attached during a dwc:Occurrence.
* Usage comments (recommendations regarding content, etc., not normative): No inference can be made from this term that a dwc:Organism interacted in any other way than being connected to some (not a particular) material resource of the substrate type (such as during parasitization or decomposition). A host-parasite relationship is better expressed as a dwc:OrganismInteraction.
* Examples (not normative): `stone`; `wall`; `decaying wood`; `wooden board fence`; `animal bones`
* Refines (identifier of the broader term this term refines; normative): none
* Replaces (identifier of the existing term that would be deprecated and replaced by this term; normative): none
* ABCD 2.06 (XPATH of the equivalent term in ABCD or EFG; not normative): none; probably inclusive into http://rs.tdwg.org/abcd/terms/Biotope 

Disclaimer: This is my first proposal to the DwC, so please forgive any faults and misunderstandings the above text might suffer of my guilt.

Sincerely,

W. Paul

**Comment by tucotuco** [comment]
No faults at all. This issue is very well done. Thanks.

**Comment by morelanneIF** [comment]
Hello, this term has been suggested a year ago. How does it stand today ? Is it planned to be added in the dwc list terms ?

**Comment by CecSve** [comment]
[environmentalMaterial](https://github.com/tdwg/dwc/issues/40) may be a broader term for substrate (i.e. _refines_, if implemented)

**Comment by tucotuco** [comment]
> Hello, this term has been suggested a year ago. How does it stand today ? Is it planned to be added in the dwc list terms ?

The term will be included in the next public review, which is not yet scheduled. My best guess is that we will be able to initiate the review at the beginning of January.

**Comment by crayfish66** [comment]
Thank you all for the "revitalizing" the issue. As far as I know my colleagues from the mycological (incl. lichenologists and "myxomycologists") universe wait for the term to be implemented (at least those interested in DwC :-) ).
I've looked through the environmentalMaterial discussion from 10 years ago ([https://github.com/tdwg/dwc/issues/40]). As far as I see, in addition to the @dr-shorthair  remark, the difference is that this cannot be feasibly vocabularized - the substrate for e.g. lichens, apart from the few general categories (as "rock", "soil", "wood") can be virtually anything, both from inanimate (from my own observations this may be bones, old car tires, bottles, styropor debris...) and animate realm (and here we have to deal with potentially hundreds of thousands of biological taxonomical names). Even types of rocks may be hundreds.

Another issue would be, if we'd want to further differentiate between substrates that are just "a place to live", more or less accidental, or a "purposedly chosen" by a organism, as a source of nutrients, shelter, anchor in an unstable environment etc. But these functions are sometimes hard to tell one from the other. If the 'substrate' term is accepted at all (which is more important), such topics may be looked upon when we see the practical implementation.

**Comment by CecSve** [comment]
> Isn't substrate a _role_ of an environmental material?
> 
> Substrate = environmental material that underlies something
> 
> Genus/differentia pattern

Yes, I guess. so. My thought was that environmentalMaterial is _ideally_ filled when the substrate is known. But I am not sure the [proposed ENVO vocabulary](https://ontobee.org/ontology/ENVO?iri=http://purl.obolibrary.org/obo/ENVO_00010483) for environmentalMaterial covers all the options highlighted by @crayfish66. If both environmentalMaterial and substrate are implemented, then it might make sense to change the definition of `substrate` to something along the lines of:

> Definition of the term (normative): Observed substrate on which the given organism grew. Please consider registering the environmentalMaterial as well, if the value falls within the controlled values of the ENVO vocabulary](https://ontobee.org/ontology/ENVO?iri=http://purl.obolibrary.org/obo/ENVO_00010483).

and vice versa changing the definition for https://github.com/tdwg/dwc/issues/40 to include substrate.

**Comment by themerekat** [comment]
Symbiota portals have long included a "substrate" field due to popular demand, so I imagine we have many users of MycoPortal and the Lichen Portal who would be supportive of this change. It has also come up in discussions with users about mistletoes and other parasitic, sessile organisms.

**Comment by jmn101** [comment]
I can add bryologists to the list of interested parties. As far as I understand it, substrate implies a more specific relationship than environmental material. This specific relationship is key for bryologists (as well as the already mentioned mycologists).

**Comment by ben-norton** [comment]
Forgive me if this is a silly question. Is "growing' required? In the definition, the term substrate is dependant upon the presence of a growing organism. What if the organism isn't growing? Rather, its just hanging out.

**Comment by tucotuco** [comment]
Hmm, interesting. Or if it was dead?

**Comment by fmjjones** [comment]
or if it was not growing as a direct result of the substrate, but because it was a handy place to attach to and catch other tasty things?

**Comment by crayfish66** [comment]
The issue raised by fmjjones has been addressed in:
Usage comments (recommendations regarding content, etc., not normative): This may _or may not_ imply that the organism influenced/took resources from the substrate (such as during parasitization or decomposition).

However, we can think on replacing "was growing" with more general "was attached to".

**Comment by jholetschek** [comment]
I'm about to create an updated version of the proposal with the agreed changes.

Currently, the definition is self-referencing since it contains the defined term. @crayfish66 Would you agree on replacing the word "substrate" with "material"?

**Comment by qgroom** [comment]
Perhaps `material or surface`

**Comment by jholetschek** [comment]
Yes, that sounds reasonable! If the term substrate is well-defined and established beyond DarwinCore, it doesn't make sense to replace it.

**Comment by crayfish66** [comment]
Sorry, it seems I misunderstood that you want to replace the very term by "material" or "surface", not the definition. We could try to eliminate the self-referencing, if this only doesn't make the definition too complicated. Self-referencing is common in other basic terms' definitions (see e.g. defs for 'language', 'sex' etc.) so by itself it doesn't seem to be disqualifying.

**Comment by ben-norton** [comment]
@crayfish66 I disagree that this is a well understood term and, therefore, should be allowed to have a self referencing definition. The current definition needs to be replaced.

**Comment by ben-norton** [comment]
@qgroom @jholetschek It's well understood in the domain for which it originates. Here, this is the biological domain. You could argue that the term only applies to biology, so the other domains aren't relevant. Again, I would disagree. Those domains need to know that their value does not belong in the field. Terms like language are universal and therefore aren't applicable to the current discussion. If we want the terms to be understood, then they need to be defined explicitly, even when the definition seems obvious. It also delineates what values belong in the field and which ones do not. Terms can mean different things to different people. So we need to make sure the terms are defined unambiguously. 
It sounds like a better-suited term is substrateDescription, rather than substrate. 
Are there types of objects that are not or cannot be a substrate?
Can a substrate be any material an organism is connected to? If a mite is attached to a shell, is the shell a substrate?

**Comment by qgroom** [comment]
The only problem is the parasite-host case, which I think should really be excluded within the definition.

**Comment by ben-norton** [comment]
@qgroom Talk about throwing a monkey wrench into the gears. That's tough. Why does it exclude parasite-host?

**Comment by gkampmeier** [comment]
Parasites can be internal or external, then which is the substrate?

**Comment by crayfish66** [comment]
@ben-norton It may be in the notes, as I wrote. 
I rely on your experience, which place you think would be the best.

**Comment by tucotuco** [comment]
The consensus version of the attributes of the new term:

* Term name (in lowerCamelCase for properties, UpperCamelCase for classes): substrate
* Term label (English, not normative): Substrate
* Organized in Class (e.g., Occurrence, Event, Location, Taxon): Occurrence
* Definition of the term (normative): A biotic or abiotic material to which an dwc:Organism is attached when a dwc:Event occurs.
* Usage comments (recommendations regarding content, etc., not normative): This may or may not imply that the organism influenced/took resources from the substrate (such as during parasitization or decomposition).  Declaration of a host-parasite relationship is better expressed as an organism interaction.
* Examples (not normative): `stone`; `wall`; `decaying wood`; `wooden board fence`; `animal bones`
* Refines (identifier of the broader term this term refines; normative): none
* Replaces (identifier of the existing term that would be deprecated and replaced by this term; normative): none
* ABCD 2.06 (XPATH of the equivalent term in ABCD or EFG; not normative): none; probably inclusive into http://rs.tdwg.org/abcd/terms/Biotope

**Comment by dr-shorthair** [comment]
Hmm. I refer you back to my [genus-differentia](https://github.com/tdwg/dwc/issues/504#issuecomment-2501956460) comment. Substrate is a _role_ of a material. Then shouldn't this be reflected in the definition somehow, either in the `Examples` or in the `Refines` clause . Not every stone, wall, etc is a substrate, only some individuals. So 'Wall-A' might act as a `Substrate` in the context of 'Event-7', but the same individual will act as (e.g.) a `barrier` in some other context. That isn't clear to me from the proposed definition.

**Comment by tucotuco** [comment]
> Hmm. See my [genus-differentia](https://github.com/tdwg/dwc/issues/504#issuecomment-2501956460) comment. Substrate is a _role_ of a material. Then shouldn't this be reflected in the definition somehow, either in the `Examples` or in the `Refines` clause . Not every stone, wall, etc is a substrate, only some individuals. So 'Wall-A' might act as a `Substrate` in the context of 'Event-7', but the same individual will act as (e.g.) a `barrier` in some other context. That isn't clear to me from the proposed definition.

Apologies to @dr-shorthair. I missed this when I summarized the issue and sent it through to Process - prepare for Executive review. Technically, I have to retract that for this round of review. Sorry to everyone else for that.

Yes, a particular stone can be observed fulfilling a substrate role during some period of time seems like a reasonable thing to posit. We can observe stone fulfilling that role in a dwc:Occurrence. But the examples suggest we don't really expect to track a particular stone as if it were a dwc:MaterialEntity record, but rather to track the type of material of which that substrate consists. I believe this was the intention with this term and a better name for it in that case might have been "substrateType". However, the definition actually suggests, though it isn't entirely clear, that in fact the particular rock might be the target of interest. If it was meant to be a particular rock, for example, the definition should probably use "dwc:MaterialEntity" rather than "material". Assuming that is not the case, the definition could be modified to make that clear - something like: 

"A type of biotic or abiotic material to which a dwc:Organism was attached during a dwc:Occurrence." 

Here I took the liberty of also substituting "dwc:Occurrence" for "dwc:Event" since this term is proposed to be a property of an dwc:Occurrence. If I am on track here, we don't have an existing term that this term could be a refinement of.

Since a change seem to be needed anyway, I would suggest also modifying the Usage comments to something like:

"No inference can be made from this term that a dwc:Organism interacted in any other way than being connected to some (not a particular) material resource of the substrate type (such as during parasitization or decomposition). A host-parasite relationship is better expressed as an OrganismInteraction."

OrganismInteraction is a class being proposed with Darwin Core Data Package (see https://gbif.github.io/dwc-dp/qrg/#OrganismInteraction), which will be the subject of the next Darwin Core public review beginning 1 September.

**Comment by kingenloff** [comment]
Not to throw a wrench in this, but... based on feedback from eLTER and my work to start mapping a small test group of their data, I identified that this may need to actually be 2 separate terms. 

I've proposed adding the terms [sampledSubstrateCategory](https://github.com/tdwg/dwc/issues/562) (which would capture the essence of this substrate term) and [sampledSubstrateLayer](https://github.com/tdwg/dwc/issues/563) (to capture where in the substrate samples were taken/observed).  Such a distinction will be important for vegetation surveys, ocean surveys, and anything taking samples from soil and ice cores.

**Comment by crayfish66** [comment]
After checking the descriptions of both your proposed terms I have an impression that they cover a separate kind of notions. The Substrate term as I understand it, is, shortly speaking, something that the given organism is _attached_ to - it's hard to be "attached" to the "ocean" (only to its bottom, reef, etc.), its water is rather an ambient medium, like the air in land habitats. Moreover, "layer" in vegetation studies (as traditional Braun-Blanquetian A, B, C, D - i.e. canopy, bush, undergrowth/herb layer, moss layer - or similar systems) does not also constitute a "substrate layer" (the substrate in these cases is soil, rock etc.). The "sampled" (not "sample") at the terms' beginning also suggests, that the substrate fragment has been collected, which is not always true when we cite the herbarium specimen, even less when we have "observation" as the eventType.

**Comment by ben-norton** [comment]
@kingenloff lol awesome.

**Comment by ben-norton** [comment]
@crayfish66 
https://vocabs.lter-europe.net/en/

**Comment by crayfish66** [comment]
@ben-norton : Thanks for the link!
As far as I see on the eLTER vocabulary (as far as I understand - the site has almost no "about" info - it's an attempt to construct a common vocabulary for environmental/ecological studies, like LTER), the "substrate" is included [there](https://vocabs.lter-europe.net/EnvThes/en/page/20909) but has 5 more or less different meanings - and only the no. 5 is concurrent with the "substrate" in the meaning of this proposal. What is worth thinking over is the synonym added there to this meaning (ie. Latin "substratum") that differentiates it from all the others (dealing more or less with (bio)chemical aspects of the environment). So may be it'd be useful to change the proposed "substrate" into "substratum" to disambiguise?

**Comment by crayfish66** [comment]
Just out of curiosity - is the next public review scheduled somewhat?...

W.

Date sent:	Mon, 16 Feb 2026 02:20:33 -0800
From:	John Wieczorek ***@***.***>
Send reply to:	tdwg/dwc ***@***.***>
To:	tdwg/dwc ***@***.***>
Copies to:	crayfish66 ***@***.***>, 
	Mention ***@***.***>
Subject:	Re: [tdwg/dwc] New Term - substrate (Issue #504)

> 
> 
>  tucotuco left a comment (tdwg/dwc#504)
> The full solution for this issue does not appear to have been resolved yet. I have added 
> the label "Unresolved" and have moved the issue to the next public review.
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you were mentioned.  

-----------------------------
Wojciech Paul, PhD
Polish Academy of Sciences
W. Szafer Institute of Botany
Lubicz 46
PL-31-512 KRAKOW, Poland
e-mail: ***@***.***
phone: [+48 12] 4241766
fax: [+48 12] 4219790
-----------------------------


-- 
 https://botany.edu.pl <https://botany.edu.pl>



Kolekcje IB PAN w 
Internecie / IB PAS collections on-line
https://rcin.org.pl/dlibra/collectiondescription/542 
<https://rcin.org.pl/dlibra/collectiondescription/542>


*Administratorem 
Pani/Pana danych osobowych jest Instytut Botaniki im. W. Szafera Polskiej 
Akademii Nauk z siedzibą przy ul. Lubicz 46, 31-512 Kraków. Na podstawie 
przepisów RODO mają Państwo prawo żądania dostępu, sprostowania, usunięcia 
lub ograniczenia dostępu do danych osobowych, a także wniesienia sprzeciwu 
o**raz wniesienia skargi do organu nadzorczego. Kontakt do Inspektora 
Ochrony Danych: ***@***.*** ***@***.***>. Pełna treść 
informacji odnośnie przetwarzania Pani/Pana danych znajduje się na stronie 
internetowej: https://botany.pl/rodo/ <https://botany.pl/rodo/>**
* 
<https://botany.pl>

**Comment by tucotuco** [comment]
Once the [DwC-DP public review and followup](https://github.com/tdwg/dwc/milestone/20) have concluded. The Darwin Core Maintenance Group will enter into a scheduled public review and release cycle. The idea is to have that be quarterly, but we'll have to see if we can sustain that pace. If all goes as planned, the DwC-DP milestone will be finished by the end of March and the [First post-DwC-DP release milestone](https://github.com/tdwg/dwc/milestone/22) will begin on 1 April and be concluded by 30 June.

**Comment by CecSve** [comment]
I notice there's now a [Sampled Substrate Category](https://gbif.github.io/dwc-dp/qrg/#Event__sampledSubstrateCategory) and [Sampled Substrate Layer](https://gbif.github.io/dwc-dp/qrg/#Event__sampledSubstrateLayer) in Event in DwC-DP coming from these two issues: https://github.com/tdwg/dwc/issues/563 and https://github.com/tdwg/dwc/issues/562.

I agree with @crayfish66 that those terms do not cover the original description proposed for this term (the substrate the organism was growing on), although it may hold the same value in some cases.

> So may be it'd be useful to change the proposed "substrate" into "substratum" to disambiguise?

I would prefer to stick to substrate, so it ties in well with the two ratified terms.

**Comment by baskaufs** [comment]
@CecSve @crayfish66 @tucotuco @morelanneIF @dr-shorthair @themerekat @ben-norton @jmn101 @fmjjones @qgroom @jholetschek @gkampmeier @kingenloff This issue was marked as "Unresolved" at the time of the initial DwC-DP public review. It is not clear to me that there has been additional discussion since then that has resulted in consensus. Am I right about that, or are there some consensus changes to the proposal that discussion participants are happy with?

If this is still unresolved, I'm going to take it out of the milestone for the current review cycle (<https://github.com/tdwg/dwc/milestone/22>) and it will get pushed to the next one.

**Comment by tucotuco** [comment]
There has been no activity that definitively resolves the pending issues as they stood on [2026-02-16](https://github.com/tdwg/dwc/issues/504#issuecomment-3907628500). There might be a question of whether there was already consensus at that time.

The latest definition and usage notes proposal is from [2026-06-09](https://github.com/tdwg/dwc/issues/504#issuecomment-2957400825). No dissent has been provided against those. We could take that as evidence of consensus and test that hypothesis in the next 30 days. If there is positive feedback (thumbs up or comments) and no alternate opinion in that time, we could take that as final consensus. Seem reasonable?

**Comment by baskaufs** [comment]
That sounds reasonable to me. I am going to label this as having consensus at the present time. If any of @CecSve @crayfish66 @morelanneIF @dr-shorthair @themerekat @ben-norton @jmn101 @fmjjones @qgroom @jholetschek @gkampmeier @kingenloff are good with the proposal as it stands, please indicate with a "thumbs up" reaction. If any have issues with the proposal as it currently stands, I will remove the consensus label and push the proposal to the next review cycle where discussion can continue.
